packages/google_maps_flutter/lib/src/cap.dart

```diff
@@ -0,0 +1,49 @@
+part of google_maps_flutter;
+
+/// Immutable cap that can be applied at the start or end vertex of a Polyline.
+class Cap {
+  const Cap._(this._json);
+
+  /// Cap that is squared off exactly at the start or end vertex of a Polyline
```

Comment:
Please refer https://www.dartlang.org/guides/language/effective-dart/documentation#do-separate-the-first-sentence-of-a-doc-comment-into-its-own-paragraph . Start with one line description, blank line and details. For this and sub sequent comments.

Reply:
@irevans Can you please work on this, as you are the author to code it would be easy for you to do. :-)

Reply:
@awazgyawali Sure, just to be clear, shall I resolve all the issues raised by @iskakaushik or are you planning on looking at some?

Reply:
I am done with some changes. WIll be pushing changes in some hours. Can you look after that?

Reply:
See commit 1455ca0